README.md

```diff
@@ -1,39 +1,61 @@
-RaceCapture/Pro Firmware
-========================
+RaceCapturePro Firmware
+=====
 
-Firmware for RaceCapture/Pro data acqusition / control / telemetry system
+The premier open source telemetry system for your race vehicle.  RaceCapturePro is a
+device that gets installed into your race vehicle and beams telemetry about your
+vehicle and driving habits in real time.  Combined with
+[RaceCapture App](https://github.com/autosportlabs/RaceCapture_App) and
+[RaceCapture Live](https://race-capture.com/) a driver and their pit crew can
+get the information and extra edge needed to win.
 
-http://www.autosportlabs.net/RaceCapture
+<!--
+Inspiration
+-----
+WE SHOULD DO THIS.
+-->
 
-See the License.txt file for more information on licensing for this software.
+# Overview
 
-= Flashing =
+As of this writing there are two versions of RaceCapture hardware in the wild.
 
-WARNING: Updating firmware restores the default configuration on RaceCapture/Pro
-Be sure to save your configuration using the Race Analyzer software.
+## MK1
 
-================================
-Windows
-================================
+MK1 was the original RaceCapturePro unit.  Based on the AT91SAM7s chipset, it provided
+a solid platform for our initial release.  It allowd for up to a 100Hz channel sampling
+rate and up to 10Hz GPS.  It also supported CAN & OBD2, various I/O, an SD card, and
+a remote GPS mouse for high quality GPS reception.  It came with a separate telemetry
+unit that would handle passing the data over the Cellular network.  It however has now
+been succedded by the next generation of RaceCapturePro
 
-Flash the firmware by double-clicking the flashRCP.bat file in Windows Explorer and follow the instructions.
+## MK2
 
-================================
-Linux
-================================
-While holding down the button on RaceCapture/Pro, plug the unit into USB. You should see 3 green LEDs light up.
+MK2 is the latest RaceCapturePro unit.  Like its successor it has many of the same features
+that were originally loved, all of which have been improved upon.  It also has some new features
+that were unavialable in the original MK1.  MK2 supports sampling rates of up to 1000 Hz per
+channel (a 10x improvement from MK1) and a GPS unit that can sample up to 50Hz (a 5x improvement
+from MK1).  It also includes a better Inertial unit, an upgraded processor (STM32 base), more RAM
```

Comment:
a GPS module capable of native 50Hz update rate....

..It also includes an improved 9 axis Inertial Motion Unit....

...an upgraded processor (STM32F407 Cortex M4F)....